Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2241,7 +2242,13 @@ - (void)_startTextManipulationsWithConfiguration:(_WKTextManipulationConfigurati
                 [wkToken setUserInfo:createUserInfo(token.info).get()];
                 return wkToken;
             });
-            return adoptNS([[_WKTextManipulationItem alloc] initWithIdentifier:String::number(item.identifier.toUInt64()) tokens:tokens.get()]);
+            StringBuilder identifierBuilder;
+            identifierBuilder.append(String::number(item.frameID.processIdentifier().toUInt64()));
+            identifierBuilder.append('-');
+            identifierBuilder.append(String::number(item.frameID.object().toUInt64()));
+            identifierBuilder.append('-');
+            identifierBuilder.append(String::number(item.identifier.toUInt64()));
```

Comment:
This can be:
```
auto identifier = makeString(item.frameID.processIdentifier(), '-', item.frameID.object(), '-', item.identifier);
```
Because `ObjectIdentifier` has a `StringTypeAdapter`, and `makeString` is more efficient than `StringBuilder`.

Reply:
Fixed.